pkg/api/networkservice/mechanisms/wireguard/constant.go

```diff
@@ -45,4 +45,8 @@ const (
 	SrcPublicKey = "src_public_key"
 	// DstPublicKey - Destination public key
 	DstPublicKey = "dst_public_key"
+	// SrcPrivateKey - Source private key
+	SrcPrivateKey = "src_private_key"
+	// DstPrivateKey - Destination private key
+	DstPrivateKey = "dst_private_key"
```

Comment:
Why do we need private keys?

Reply:
@glazychev-art My response would be a bit stronger: why are we sending private keys with the message?  That would seem to abrogate all of the security of wireguard.

Reply:
@edwarnicke You are absolutely right. But this code is outdated. I removed this mistake.